.github/blunderbuss.yml

```diff
@@ -0,0 +1,11 @@
+assign_issues:
+  - dinagraves
```

Comment:
Should this be assigned to a team?

Can you also update the CODEOWNERS file as well?

Reply:
Hi Kurtis, I've modified the files. PTAL if the assignments are okay.

Reply:
it looks like owlbot has reverted the changes. @busunkim96 how do we typically handle this for these repos?

Reply:
I think `@googleapis/python-samples-reviewers` is the correct owner for samples. I've provided write access for `@googleapis/python-samples-reviewers`.

Reply:
I don't think any owlbot changes are needed. PTAL

Reply:
Ditto what @parthea said, it doesn't look like the blunderbuss config is being reverted to me.

I had GitHub's code review auto-assignment turned on in `python-samples-owners` instead of blunderbuss, but that might not be an option anymore if we can't access team settings... In that case we should roll this out as a new templated file in synthtool.